modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptFetchClientCodegen.java

```diff
@@ -56,6 +55,7 @@ public TypeScriptFetchClientCodegen() {
 
         this.cliOptions.add(new CliOption(NPM_REPOSITORY, "Use this property to set an url your private npmRepo in the package.json"));
         this.cliOptions.add(new CliOption(WITH_INTERFACES, "Setting this property to true will generate interfaces next to the default class implementations.", SchemaTypeUtil.BOOLEAN_TYPE).defaultValue(Boolean.FALSE.toString()));
+        this.cliOptions.add(new CliOption(USE_SINGLE_REQUEST_PARAMETER, "Setting this property to true will generate functions with a single parameter containing all parameters", SchemaTypeUtil.BOOLEAN_TYPE).defaultValue(Boolean.FALSE.toString()));
```

Comment:
Maybe we can phrase this a little bit clearer?

e.g. 
`Setting this property to true will generate functions with a single argument containing all API endpoint parameters instead of one argument per parameter.`

Reply:
Do you want me to change the name of the option?
I see in other thread about this that some people call it "GroupParameter" or "CollectParameter"?
That would require change everywhere in my PR.